Lm docs (#2267)
kyma-project/control-plane: docs/kyma-environment-broker/03-01-service-description.md

  <summary label="own_cluster-plan">
  Own cluster plan
  </summary>

> </details>
> </div>

Reviewer: ```suggestion
```

| Parameter name | Type | Description | Required | Default value |
| ---------------|-------|-------------|----------|---------------|
| **kubeconfig** | string | Kubeconfig that points to the cluster where you install Kyma. | Yes | None |
| **shootDomain** | string | Domain of the shoot where you install Kyma. | Yes | None |
| **shootName** | string | Name of the shoot where you install Kyma. | Yes | None |

> 

Reviewer: ```suggestion

</details>
</div>

```

## Preview cluster plan

> Preview plan allows to test an integration with Lifecycle Manager. The preview plan skips steps which integrates Kyma Environment Broker and Reconciler.

Reviewer: ```suggestion
The preview plan allows to test integration with Lifecycle Manager. The preview plan skips steps which integrate Kyma Environment Broker and Reconciler.
```

### Provisioning parameters

These are the provisioning parameters for the `preview` plan that you configure:

<div tabs name="own_cluster-plan" group="own_cluster-plan">
  <details>
  <summary label="own_cluster-plan">
  Preview cluster plan
  </summary>

> </details>
> </div>

Reviewer: ```suggestion
```

| Parameter name | Type | Description | Required | Default value |
| ---------------|-------|-------------|:----------:|---------------|
| **machineType** | string | Specifies the provider-specific virtual machine type. | No | `m5.2xlarge` |
| **volumeSizeGb** | int | Specifies the size of the root volume. | No | `50` |
| **region** | string | Defines the cluster region. | No | `westeurope` |
| **zones** | string | Defines the list of zones in which Runtime Provisioner creates a cluster. | No | `["1"]` |
| **autoScalerMin[<sup>1</sup>](#update)** | int | Specifies the minimum number of virtual machines to create. | No | `3` |
| **autoScalerMax[<sup>1</sup>](#update)** | int | Specifies the maximum number of virtual machines to create, up to `40` allowed. | No | `10` |
| **maxSurge[<sup>1</sup>](#update)** | int | Specifies the maximum number of virtual machines that are created during an update. | No | `4` |
| **maxUnavailable[<sup>1</sup>](#update)** | int | Specifies the maximum number of virtual machines that can be unavailable during an update. | No | `1` |

> 

Reviewer: ```suggestion

</details>
</div>

```

| **shootDomain** | string | Domain of the shoot where you install Kyma. | Yes | None |
| **shootName** | string | Name of the shoot where you install Kyma. | Yes | None |

> ## Preview cluster plan

Reviewer 2: I dont know why, but when previewing this file I don't see this section see: https://github.com/kyma-project/control-plane/blob/d089a040db5435cb9e53585b26a6188e68dfe5d2/docs/kyma-environment-broker/03-01-service-description.md

Author: fixed
